synctl/cli.py

```diff
@@ -2198,7 +2203,7 @@ def __get_max_lo_label_length(self, loc_list, max_len=60):
         max_result["max_display_label"] = display_label_len if display_label_len < 60 else 60
         return max_result
 
-    def print_a_location_details(self, location_id, single_location, show_details=False):
+    def print_a_location_details(self, location_id, single_location, show_details=False, show_json=False):
         """show a Synthetic location details data"""
         if single_location is None or len(single_location) == 0 or location_id is None:
             print("no Synthetic location")
```

Comment:
Issue: This message is misleading, I think it should be updated to something like: Detail data is only available for a selected location. Usage: synctl get location <location-id> --show-details or synctl get location <location-id> ----show-json